tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -257,6 +257,12 @@ static int test_wdt_callback_1(void)
 		}
 	}
 
+	err = wdt_disable(wdt);
+	if (err < 0 && err != -EPERM) {
```

Comment:
`&& err != -EFAULT` should be added here. Otherwise, this will fail for example on nRF5340 where the watchdog can be disabled but is not automatically started after reset.

Reply:
Done!